neuralmonkey/tf_manager.py

```diff
@@ -88,7 +88,7 @@ def __init__(self,
         init_op = tf.global_variables_initializer()
         for sess in self.sessions:
             sess.run(init_op)
-        self.saver = tf.train.Saver(max_to_keep=self.saver_max_to_keep,
+        self.saver = tf.train.Saver(max_to_keep=None,
```

Comment:
tohle bylo podle mě blbě, když to tam keepovalo track o počtu uloženejch proměnnejch na několika místech..

Reply:
Pokud to chapu, tak s max_to_keep=None se ti budou ukladat nove a nove checkpointy. Nebo mame nekde jinde implementovanou logiku, ktera to promazava?

Reply:
Právě že tam ta logika je - opice si drží seznam souborů a odpovídající validační skóre a při uložení se vždycky přepíše soubor s nejhoršíma proměnnýma.

Reply:
.. a ten seznam má velikost `self.saver_max_to_keep`